site/_posts/2016-11-18-sprint-49-review-november-16-2016.html.md

```diff
@@ -0,0 +1,146 @@
+
+The theme of this sprint is bug fixing. Of the 354 pull requests merged to the main [ManageIQ repo](https://github.com/ManageIQ/manageiq), more than half were fixing and patching stuff. As we're in stabilization mode with the Euwe GA in sight, this is only natural. However we still managed 54 enhancement PRs, and backported 162 to Euwe branch. Combined with 118 PRs from integration_tests, and another 120 from Service UI, the website, and various providers etc., we amassed 592 PRs altogether from [the different repos](https://github.com/manageiq)!
+
+Let's take a look at some of the things that were fixed:
+
+* Chargeback: during the last sprint, we added the weekly/monthly rates for 'per time' types but missed the daily ones, now daily rates have been reinstated
+* Central Admin: squashed a bug where the ServiceOrder was not created on the remote region and it prevented service template provisioning from working end to end; VM reconfigure is also fixed to have proper routing to remote regions
+* Automate: key/pair attributes fixed to match Simulation for Automate scheduling work done in the previous sprint
+* Git Automate: now repos without the domain directory can be supported; plus refresh cleanup on the contents and references where new branches and tags were not getting fetched
+* UI: cleanup of Charting/Topology/Toolbar buttons, as many of these pieces were replaced
+* UI: fixed regression from VM tree changes - a performance enhancement piece from last sprint by hiding VMs/instances from explorer trees by default, which caused some issues as the code was dependent on the tree selection of VMs
+* I18N: getting the missing translations localized
+* Hawkular: professional bug buster with 18 of the 19 PRs towards fixes, including 3 blockers
+* QE: lots of bug fixing means keeping busy with lots of bug verifications
+
+Nonetheless, there's always room for improvements, and here are some highlights:
+
+* Amazon: moved Cloud manager refresh to new DTO (data transfer object) method thus increasing the saving efficiency
+* RHEV: piggybacking on the work - targeted refresh when provisioning - from previous sprint, time taken for provisioning a new VM to a large environment reduced from 17 min to 1 min
+* RHEV: added live migration of VM, and the ability to import VMs from VMware to RHEV (also an API enhancement)
+* VMware vCloud: XLAB team implemented the feature where credentials for vCloud events can now be added in the UI instead of using the rails console, improving the user experience
+* Performance: there's a shiny new manageiq-performance gem for automating and collecting of metrics with initial focus on UI performance, it's emphasis is on being modular and configurable so as to unify the toolchain for everyone (QE, performance team, developers, support)
```

Comment:
Typo `it's` -> `its`.  Probably should be a semicolon or new sentence there as well because it's a run-on sentence.